util/stmtsummary/statement_summary.go

```diff
@@ -631,6 +700,7 @@ func newStmtSummaryByDigestElement(sei *StmtExecInfo, beginTime int64, intervalS
 		firstSeen:    sei.StartTime,
 		lastSeen:     sei.StartTime,
 		backoffTypes: make(map[fmt.Stringer]int),
+		isInternal:   sei.IsInternal,
```

Comment:
It's internal only when all of its statements are internal, not just the first one.